release/rotation

```diff
@@ -2,6 +2,7 @@
 # to access an ad-hoc machine for Windows testing. This can be done by opening a PR
 # like the following: https://github.com/DACH-NY/daml-language-ad-hoc/pull/37
 U0PNNPY05 robin-da
+U03518HUVBP stewartstewart-da
```

Comment:
we do think it's good to do that fairly quickly after joining, but I figured _tomorrow_ was perhaps a bit too harsh with the 2.0 thing going, so left @robin-da in charge of that.